### DISCUSSION TITLE
ELI5 basic connector setup

### DISCUSSION THREAD

**Discussion by djdule** [International-Data-Spaces-Association/DataspaceConnector#58] [Q&A] [answered]
Hi all,

Can someone help me please with basic configuration of connectors. I am someone coming from ops background so building connectors was straightforward. However I spent lots of time reading about how to make it run and I still did not manage it. Can someone help me with basic steps like:

-creating resource on connector A, for example uploading simple file
-consuming that resource from connector B, having that file in other connector
-seeing logs or any kind of information that transfer happened

Thanks a lot in advance !

**Comment by juliapampus** [comment, marked as answer]
Hi, thanks for your questions. Have you already had a look at the wiki? There are some guides for [setting up the connector](https://github.com/FraunhoferISST/DataspaceConnector/wiki/Development) and [managing resources](https://github.com/FraunhoferISST/DataspaceConnector/wiki/IDS-Communication-Guide). If your problems remain, feel free to contact us here again.

**Reply by djdule** [reply]
Yes, I did tried that example. I managed to set up connector and create resource. What confuses me is it accepts only string as data, whereas I wanted to try out how it would work with files, for example JPG picture.

**Reply by juliapampus** [reply]
I am happy to hear it is working. The connector only sends strings via a https connection. As all files have to be encrypted for that sooner or later, the connector currently only provides an endpoint for retreiving strings. It is not encoding files itself, a service for automating interaction with the connector has to do that. When the filetype in the metadata is set correctly, the consumer then is able to decode this string to a file again.
